.github/workflows/validate-workflow.yml

```diff
@@ -22,6 +22,11 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
 
+    - name: Reconfigure git to use HTTP authentication
```

Comment:
was running into an issue with the `node-bigcommerce` package not being configured for ssh... this seems to be a workaround - https://github.com/actions/setup-node/issues/214#issuecomment-810829250.

Will also hitup Booker to see if he has any insight since he created the repo

Reply:
Curious, have we tried appending `github:` to the repo reference?
```json
{
  "node-bigcommerce": "github:bigcommerce/node-bigcommerce",
}
```